<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWinScreenEdgesConfigForm</class>
 <widget class="QWidget" name="KWinScreenEdgesConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>511</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Active Screen Edge Actions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="KWin::Monitor" name="monitor" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>To trigger an action push your mouse cursor against the edge of the screen in the action's direction.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Window Management</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="quickMaximizeBox">
        <property name="text">
         <string>Maximize windows by dragging them to the top of the screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="quickTileBox">
        <property name="text">
         <string>Tile windows by dragging them to the side of the screen</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Quarter tiling triggered in the outer</string>
          </property>
          <property name="buddy">
           <cstring>electricBorderCornerRatio</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="electricBorderCornerRatio">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>49</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>of the screen</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Other Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="desktopSwitchLabel">
        <property name="toolTip">
         <string>Change desktop when the mouse cursor is pushed against the edge of the screen</string>
        </property>
        <property name="text">
         <string>&amp;Switch desktop on edge:</string>
        </property>
        <property name="buddy">
         <cstring>desktopSwitchCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="desktopSwitchCombo">
        <item>
         <property name="text">
          <string comment="Switch desktop on edge">Disabled</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Only When Moving Windows</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always Enabled</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>Amount of time required for the mouse cursor to be pushed against the edge of the screen before the action is triggered</string>
        </property>
        <property name="text">
         <string>Activation &amp;delay:</string>
        </property>
        <property name="buddy">
         <cstring>activationDelaySpin</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KIntSpinBox" name="activationDelaySpin">
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Amount of time required after triggering an action until the next trigger can occur</string>
        </property>
        <property name="text">
         <string>&amp;Reactivation delay:</string>
        </property>
        <property name="buddy">
         <cstring>triggerCooldownSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KIntSpinBox" name="triggerCooldownSpin">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KWin::Monitor</class>
   <extends>QWidget</extends>
   <header>monitor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>monitor</tabstop>
  <tabstop>desktopSwitchCombo</tabstop>
  <tabstop>activationDelaySpin</tabstop>
  <tabstop>triggerCooldownSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quickTileBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quickTileBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>electricBorderCornerRatio</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quickTileBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
